contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+# Proposal to Standardize Development Process
+
+## Links
+```
+---
+Links:
+- Issues:
+  - [someIssueURL]()
+- PRs:
+  - https://github.com/kubernetes/community/pull/967
+- MailingListDiscussions:
+  - https://groups.google.com/forum/#!topic/kubernetes-dev/65A-3ULYPB0
+- Documentation:
+  - [someDocsLinkURL]()
+```
+
+## Responsible SIG(s)
+
+- SIG Release
+- SIG PM
+- SIG Architecture
+- SIG Governance
+- Technical Steering Committee
+
+## Summary
+
+A standardized development process for Kubernetes is proposed in order to:
+- provide a common structure for proposing changes to Kubernetes
```

Comment:
I definitely think that's a good point about work bypassing any process in order to follow the path of least resistance. I am hoping that SIGs will find the process useful enough to push back against exceptions on their own accord. Once we've decomposed the monorepo I think there will be a natural desire at the SIG level to enforce some sort of structure around change management in order to manage their own exposure to risk